corehq/util/context_processors.py

```diff
@@ -99,6 +99,12 @@ def enterprise_mode(request):
     }
 
 
+def skip_google_tag_manager(request):
+    return {
+        'skip_google_tag_manager': settings.SERVER_ENVIRONMENT in ['icds', 'enikshay']
```

Comment:
@proteusvacuum @snopoke  is there a reason why enikshay isn't in ENTERPRISE_MODE ? seemed like there was some error https://github.com/dimagi/commcare-hq/pull/16985#issuecomment-315352298 any idea if that was resolved?

Reply:
enikshay can use enterprise mode. We couldn't when we were on softlayer since it applies to the entire HQ instance so we add a specific check from the domain name: https://github.com/dimagi/commcare-hq/pull/16985/commits/60d4bda62c94ea93867c4992ef05bbd06c2c7ec2

Now that it's on a separate cluster we can set enterprise mode